vnext/Scripts/UnitTest.ps1

```diff
@@ -5,12 +5,23 @@
 #
 param (
 	[ValidateSet('x64', 'x86')]
-	$Platform = 'x64',
+	[string] $Platform = 'x64',
+
 	[ValidateSet('Debug', 'Release')]
-	$Configuration = 'Debug',
+	[string] $Configuration = 'Debug',
```

Comment:
>$Configuration [](start = 10, length = 14)

Shouldn't we also test ship at some point? Why is Configuration always Debug?

Reply:
This doesn't mean 'always Debug'. This means 'default to Debug'.
You can always override to release and/or x86.
Even better, PowerShell allows you to browse these values by just typing TAB.